### PR TITLE
Add doctor page

### PR DESCRIPTION
**Pull request by mohannadhanafi**
Relates #74

### CODE REVIEW

File: src/controllers/addDoctor.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+exports.get = (req, res) => {
+  res.render('addDoctor', {
+    title: 'Add Doctor',
```
[the remark this answers is not in the document]
Comment (mohannadhanafi) [reply src/controllers/addDoctor.js]: Thank you @MohammedYehia for your tips :heart:

File: src/controllers/index.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ const articles = require('./articles');
 const doctors = require('./doctors');
 const signUp = require('./signUp');
 const signIn = require('./signIn');
+const doctor = require('./doctor');
```
[the remark this answers is not in the document]
Comment (mohannadhanafi) [reply src/controllers/index.js]: It exists in controller folder

[the remark this answers is not in the document]
Comment (MohammedYehia) [reply src/controllers/index.js]: check them again

File: src/controllers/admin.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 const { insertArticle } = require('../model/queries/addData');
 
 exports.get = (req, res) => {
-  console.log(res.locals.unlockCookie);
   if (res.locals.unlockCookie.permission === 'admin') {
```
[the remark this answers is not in the document]
Comment (MohammedYehia) [reply src/controllers/admin.js]: @mohannadhanafi

File: src/controllers/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mohannadhanafi) [reply src/controllers/index.js]: I uploaded it now :smile: 
sorry :+1:

**Pull request merged**